pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -1017,13 +1026,29 @@ class Build {
                             // Issue: https://issues.jenkins.io/browse/JENKINS-64779
                             if (context.WORKSPACE != null && !context.WORKSPACE.isEmpty()) {
                                 if (cleanWorkspaceAfter) {
-                                    context.println "Cleaning workspace files: " + context.WORKSPACE + "/*"
+                                    context.println "Cleaning workspace non-hidden files: " + context.WORKSPACE + "/*"
                                     context.sh(script: "rm -rf " + context.WORKSPACE + "/*")
+
+                                    // Clean remaining hidden files using cleanWs
+                                    try {
+                                        context.println "Cleaning workspace hidden files using cleanWs: " + context.WORKSPACE
+                                        context.cleanWs notFailBuild: true, disableDeferredWipeout: true, deleteDirs: true
+                                    } catch (e) {
+                                        context.println "Failed to clean ${e}"
+                                    }
                                 } else if (cleanWorkspaceBuildOutputAfter) {
-                                    context.println "Cleaning workspace build output files: " + context.WORKSPACE + "/workspace/build/src/build"
+                                    context.println "Cleaning workspace build output non-hidden files: " + context.WORKSPACE + "/workspace/build/src/build"
```

Comment:
Why are these necessary? We're not using `/*` on these so the `rm -rf ` shoulkd be sufficient

Reply:
so you mean, just use the above rm -rf ?
since there should not be any hidden files in the build output?
so remove the cleanWs for this bit?

Reply:
oh ! Doh! I see what you mean :-) (slow today!)

Reply:
removed